fsevents.js

```diff
@@ -6,7 +6,11 @@
 /* jshint node:true */
 'use strict';
 
-var Native = require('./build/Release/fse');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname,'./package.json')));
+var Native = require(binding_path);
```

Comment:
> Windows accepts forwards slashes in paths just fine

I'm aware of that from my work with build tooling, but I have found that not being diligent with `path.join` causes slash confusion to bite you when you don't expect it. Also, more straightforward to just use it than test every case on windows to make sure.